<launch>
    <include file="$(find infinity_uwb_ros)/launch/uwb_node.launch">
        <arg name="serial_name" value="/dev/tty.SLAB_USBtoUART" />
        <arg name="baudrate" value="921600"/>
    </include>
    <node pkg="swarm_drone_proxy" name="swarm_drone_proxy" type="swarm_drone_proxy_node" output="screen"> 
        <param name="force_self_id" value="7" type="int" />
        <param name="vins_topic" value="/vins_estimator/imu_propagate" type="string" />
    </node>
    <node pkg="swarm_vo_fuse" name="swarm_vo_fuse" type="swarm_vo_fuse_node" output="screen">
       <param name="force_freq" value="5" type="double" />
       <param name="max_accept_cost" value="1.0" type="double" />
       <param name="max_keyframe_num" value="100" type="int" />
       <param name="min_keyframe_num" value="10" type="int" />
       <param name="thread_num" value="4" type="int" />
    </node>
    <!-- <node pkg="swarm_vo_fuse" name="swarm_vo_visualize" type="swarm_vo_visualize.py" output="screen">  -->
    <!-- </node> -->
    <node pkg="swarm_vo_fuse" name="swarm_fix_odom" type="swarm_fix_odom.py" output="screen"> 
    </node>
</launch>